pkg/config/templates/wicked-ifcfg-bond-master

```diff
@@ -0,0 +1,19 @@
+STARTMODE='onboot'
+BONDING_MASTER='yes'
+BOOTPROTO='{{ .Method }}'
+{{ if eq .Method "static" -}}
+IPADDR={{ .IP }}
+NETMASK={{ .SubnetMask }}
+{{- end }}
+{{ range $i, $interface := .Interfaces -}}
+BONDING_SLAVE_{{ $i }}='{{ $interface.Name }}'
+{{ end }}
+{{  with .BondOption -}}
+{{- $bondMode := or .Mode "balance-tlb" -}}
```

Comment:
We have a bit of inconsistency here:
- From installer,`balance-rr` is used by default (I think because it's the first option)
- From PXE boot, if no bond mode id specified, then `balance-tlb` is used.

Reply:
Folks, which should be the default? Do you have any preference for the order of bond mode options?

Reply:
I think `blancer-rr` is good for general purposes but must be configured from Switch to support static trunking first, however, the `balance-tlb` does not require any Switch configs so let's just default to `balance-tlb` for now.
http://qnapsupport.net/load-balance-setup-which-is-the-best-port-trunking-option/